docker-compose.yml

```diff
@@ -52,16 +52,26 @@ services:
             timeout: 10s
             retries: 10
     postgres:
-        image: postgres:11
+        image: 'postgres:latest'
+        ports:
+            - '${POSTGRES_PORT:-5432}'
         environment:
-            POSTGRES_PASSWORD: password
-            POSTGRES_USER: root # superuser name
+            PGPASSWORD: '${POSTGRES_PASSWORD:-postgres}'
+            PGPORT: '${POSTGRES_PORT:-5432}'
             POSTGRES_DB: main
+            POSTGRES_USER: '${POSTGRES_USER:-postgres}'
+            POSTGRES_PASSWORD: '${POSTGRES_PASSWORD:-postgres}'
         healthcheck:
-            test: ["CMD-SHELL", "pg_isready -U postgres"]
-            interval: 10s
+            test:
+                - CMD
+                - pg_isready
+                - '-q'
+                - '-d'
+                - '${DB_DATABASE}'
+                - '-U'
+                - '${DB_USERNAME}'
+            retries: 3
```

Comment:
Can you provide a basic rundown of what these changes do/why they're needed

Reply:
I completely reverted the healthcheck changes and things seem to be working fine. So in the end, these changes aren't needed